### PR TITLE
cylc vip

### PR DESCRIPTION
**Pull request by wxtim**
**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

Companion to https://github.com/cylc/cylc-flow/pull/5121

### CODE REVIEW

File: src/user-guide/compound-commands.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.. _CompoundCommands:
+
+Compound Commands
+=================
+
+Cylc Provides compound commands which carry out more than one
```
Comment (datamel) [inline src/user-guide/compound-commands.rst:6]:
```suggestion
Cylc provides compound commands which carry out more than one
```

File: src/user-guide/compound-commands.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.. note::
+
+   Use ``cylc command --help`` to get help for each compound command,
+   including a full list of available options.
+
+
+``cylc vip`` (Validate, Install and Play)
+-----------------------------------------
+
+Cylc VIP is the same as running:
```
Comment (datamel) [inline src/user-guide/compound-commands.rst:21]: Could we give the path as explained in the example?

**Review by datamel — APPROVED** [review]
I'm back on your vip branch in `cylc-flow` so thought I'd review this at the same time. Looks good, a couple of minor comments.

File: src/user-guide/compound-commands.rst (hunk 2 above)
Comment (wxtim) [reply src/user-guide/compound-commands.rst:21]: like this?
```suggestion
``cylc vip /home/me/cylc-src/my-workflow`` is the same as running:
```

File: src/7-to-8/major-changes/cli.rst [hunk 3]
```diff
@@ -113,6 +113,13 @@ GUI. Consequently the following commands have also been removed:
 The ``cylc gui`` command remains, it launches a standalone version of the
 web GUI (providing the `Cylc UI Server`_ is installed).
 
+.. seealso::
+
+   :ref:`CompoundCommands` - For your convenience we have
+    provided commands which represent common working practices. For
+    example ``cylc vip``(Validate-Install-Play) which runs
```
Comment (MetRonnie) [inline src/7-to-8/major-changes/cli.rst:120]:
```suggestion
    example ``cylc vip`` (Validate-Install-Play) which runs
```

File: src/7-to-8/cheat-sheet.rst [hunk 4]
```diff
@@ -59,6 +59,9 @@ Install a workflow from source and run it:
          # run the installed workflow
          cylc play <name>
 
+         # alternatively:
```
Comment (MetRonnie) [inline src/7-to-8/cheat-sheet.rst:62]:
```suggestion
         # alternatively:
         # validate, install & play
```

File: src/7-to-8/cheat-sheet.rst [hunk 5]
```diff
@@ -59,6 +59,9 @@ Install a workflow from source and run it:
...
+         cylc vip .
```
Comment (MetRonnie) [inline src/7-to-8/cheat-sheet.rst:63]:
```suggestion
         # alternatively:
         # validate, install & play
         cylc vip <name>
```

**Review by MetRonnie — APPROVED** [review]

**Pull request merged**